geth/account/accounts.go

```diff
@@ -237,17 +246,20 @@ func (m *Manager) SelectAccount(address, password string) error {
 
 // SelectedAccount returns currently selected account
 func (m *Manager) SelectedAccount() (*SelectedExtKey, error) {
+	m.mu.RLock()
+	defer m.mu.RUnlock()
+
 	if m.selectedAccount == nil {
 		return nil, ErrNoAccountSelected
 	}
 	return m.selectedAccount, nil
 }
 
 // Logout clears selectedAccount.
-func (m *Manager) Logout() error {
+func (m *Manager) Logout() {
+	m.mu.Lock()
+	defer m.mu.Unlock()
 	m.selectedAccount = nil
```

Comment:
Isn't there a new line missing here?

Reply:
Between `defer m.mu.Unlock()` and `m.selectedAccount = nil`?

Reply:
Yes. That block always seems to be a header everywhere else, so might as well keep the same style here.